chapters/en/chapter5/evaluation.mdx

```diff
@@ -0,0 +1,387 @@
+
+```python
+from evaluate import load
+
+wer_metric = load("wer")
+
+wer = wer_metric.compute(references=[reference], predictions=[prediction])
+
+print(wer)
+```
+**Print Output:**
+```
+0.3333333333333333
+```
+
+0.33, or 33.3%, as expected! We now know what's going on under-the-hood with this WER calculation.
+
+Now, here's something that's quite confusing! What do you think the upper limit of the WER is? You would expect it to be
+1 or 100% right? Nuh uh! Since the WER is the ratio of errors to number of words (N), there is no upper limit on the WER!
+Let's take an example were we predict 10 words and the target only has 2 words. If all of our predictions were wrong (10 errors),
+we'd have a WER of 10 / 2 = 5, or 500%! This is something to bear in mind if you train an ASR system and see a WER of over
+100%. Although if you're seeing this, something has likely gone wrong... 😅
+
+## Word Accuracy
+
+We can flip the WER around to give us a metric where _higher is better_. Rather than measuring the word error rate,
+we can measure the _word accuracy (WAcc)_ of our system:
+
+\begin{equation}
+WAcc = 1 - WER \nonumber
+\end{equation}
+
+The WAcc is also measured on the word-level, it's just the WER reformulated as an accuracy metric rather than an error
+metric. The WAcc is very infrequently quoted in the speech literature - we think of our system predictions in terms of
+word errors, and so prefer error rate metrics that are more associated with these error type annotations.
+
+## Character Error Rate
+It seems a bit unfair that we marked the entire word for "sit" wrong when in fact only one letter was incorrect!
+That's because we were evaluating our system on the word level, thereby annotating errors on a word-by-word basis.
+The _character error rate (CER)_ assesses systems on the _character level_. This means we divide up our words into their
+individual characters, and annotate errors on a character-by-character basis:
+
+| Reference:  | t   | h   | e   |     | c   | a   | t   |     | s   | a     | t   |     | o   | n   |     | t   | h   | e   |     | m   | a   | t   |
+|-------------|-----|-----|-----|-----|-----|-----|-----|-----|-----|-------|-----|-----|-----|-----|-----|-----|-----|-----|-----|-----|-----|-----|
+| Prediction: | t   | h   | e   |     | c   | a   | t   |     | s   | **i** | t   |     | o   | n   |     | t   | h   | e   |     |     |     |     |
+| Label:      | ✅   | ✅   | ✅   |     | ✅   | ✅   | ✅   |     | ✅   | S     | ✅   |     | ✅   | ✅   |     | ✅   | ✅   | ✅   |     | D   | D   | D   |
+
+We can see now that for the word "sit", the "s" and "t" are marked as correct! It's only the "i" which is labelled as a
+substitution error (S). Thus, we reward our system for the partially correct prediction 🤝
+
+In our example, we have 1 character substitution, 0 insertions, and 3 deletions. In total, we have 14 characters. So, our CER is:
+
+\begin{aligned}
```

Comment:
Same as earlier: the formula doesn't render

Reply:
Resolved in https://github.com/huggingface/audio-transformers-course/pull/70/commits/2d0ab9194c416f9ea4b0e78e68311e9b02332e65